scripts/publish.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+packages=(
+    "opentelemetry"
+    "opentelemetry-http"
+    "opentelemetry-semantic-conventions"
+    "opentelemetry-jaeger-propagator"
+    "opentelemetry-sdk"    
+    "opentelemetry-proto"
+    "opentelemetry-otlp"
+    "opentelemetry-stdout"
+    "opentelemetry-jaeger"
+    "opentelemetry-zipkin"
+    "opentelemetry-prometheus"
+    "opentelemetry-appender-log"
+    "opentelemetry-appender-tracing"
+
+    # Add more packages as needed, in the right order. A package should only be published after all it's dependencies have been published
+)
+
+# Set the current directory to one level above the scripts directory
+current_dir=$(pwd)/..
+cd "$current_dir"  # Change to the current directory
+
+# Iterate over the list of packages
+for package in "${packages[@]}"; do
+    if [ -d "$package" ]; then
+        echo "=================================================="
+        echo "Processing package: $package"
+        cd "$package"  # Change to the directory of package
+
+        # Extract the name and version from Cargo.toml
+        name=$(grep -m1 '^name =' Cargo.toml | cut -d'"' -f2)
+        version=$(grep -m1 '^version =' Cargo.toml | cut -d'"' -f2)
+
+        if [[ -n "$name" && -n "$version" ]]; then
+            echo "Found package $name with version $version" in cargo.toml
+            
+            # Tag the version in git
+            tag="${name}-${version}"
+            tag_message="${name} ${version} release."
+            # uncomment the following lines after verifying all looks good.
+            # git tag -a "$tag" -m "\"$tag_message\""
+            # git push origin "$tag"
```

Comment:
Script can take a mandatory parameter for dry-run and publish

```bash
$ publish --dryrun # will do dry-run (or make it default)
$ publish --publish # will do publish
```

This way, no need to uncomment lines for publishing :)

Reply:
dryrun will fail anyway as dependencies are not going to be there. The commented out is just to prevent accidentally running and causing issues!

Just wanted to have something simple, as long term, we could use something like https://github.com/MarcoIeni/release-plz